.github/release-drafter.yaml

```diff
@@ -0,0 +1,61 @@
+name-template: "v$RESOLVED_VERSION"
+tag-template: "v$RESOLVED_VERSION"
+categories:
+  - title: Breaking Changes 🛠
+    labels:
+      - breaking
+  - title: 🏕 Features
+    labels:
+      - feature
+  - title: Fixes 🔧
+    labels:
+      - fix
+  - title: Other Changes
+    labels:
+      - chore
+change-template: "- $TITLE (#$NUMBER)"
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
+autolabeler:
+  - label: "build"
+    title:
+      - '/build(\(.*\))?:/i'
+  - label: "ci"
+    title:
+      - '/ci(\(.*\))?:/i'
+  - label: "documentation"
+    title:
+      - '/docs(\(.*\))?:/i'
+  - label: "fix"
+    title:
+      - '/fix(\(.*\))?:/i'
+  - label: "chore"
+    title:
+      - '/chore(\(.*\))?:/i'
+  - label: "feature"
+    title:
+      - '/feat(\(.*\))?:/i'
+  - label: "breaking"
+    title:
+      - "/.*!:/i"
+replacers:
+  - search: '/(fix|chore|feat)(\(.*\))?(!)?:/gi' # Replace any supported starting tag and scope with empty string
```

Comment:
the comment implies this should be all our supported starting tags, but only has a few - should this be more comprehensive?